app/validations/household_validations.rb

```diff
@@ -56,6 +51,21 @@ def validate_household_pregnancy(record)
     end
   end
 
+  def validate_household_number_of_other_members(record)
+    (2..8).each do |n|
+      validate_person_age(record, n)
+      validate_person_age_matches_economic_status(record, n)
+      validate_person_age_matches_relationship(record, n)
+      validate_person_age_and_gender_match_economic_status(record, n)
+      validate_person_age_and_relationship_matches_economic_status(record, n)
+    end
+    validate_partner_count(record)
+  end
+
+  def validate_person_1_age(record)
```

Comment:
Think Katrina's right this also needs to validate `validate_person_age_matches_economic_status` for person 1